conans/client/printer.py

```diff
@@ -26,11 +27,17 @@ def __init__(self, out):
     def print_inspect(self, inspect):
         for k, v in inspect.items():
             if isinstance(v, dict):
-                self._out.writeln("%s" % k)
+                self._out.writeln("%s:" % k)
                 for sk, sv in sorted(v.items()):
                     self._out.writeln("    %s: %s" % (sk, str(sv)))
             else:
-                self._out.writeln("%s: %s" % (k, str(v)))
+                if '\n' in str(v):
```

Comment:
Relying on this condition seems a bit weak. What about options declared as tuple?

Reply:
Please include tests with different types of declared options and default_options

Reply:
Yes! Thanks

Reply:
BTW, much better your option using instanceof tuple :smile_cat: thanks!

Reply:
Unfortunately, won't work for Python 2.7. The condition is about multiline statements. I didn't find a smart way to solve it using tuple condition, maybe we should revert and check for '\n' again.

Reply:
exactly, as I said, that condition is about multiple lines, regardless of instance type. checking for '\n' could be weak, but fits better than by type, because I'm trying to format that string when it contains multiple lines only.

the latest commit uses tuple as condition and doesn't work, but one commit before was working for both python versions.